src/cascadia/TerminalApp/ShortcutActionDispatch.cpp

```diff
@@ -19,6 +19,27 @@ using namespace winrt::TerminalApp;
 
 namespace winrt::TerminalApp::implementation
 {
+    std::wstring ExtractAction(const ActionAndArgs& actionAndArgs)
+    {
+        std::wstring id{ actionAndArgs.GenerateID() };
+        const auto segment1 = id.find(L'.');
+        const auto segment2 = id.find(L'.', segment1 + 1);
+        if (segment1 != std::wstring::npos)
+        {
+            if (segment2 != std::wstring::npos)
+            {
+                return id.substr(segment1 + 1, segment2 - segment1 - 1);
+            }
+            else
+            {
+                return id.substr(segment1 + 1);
+            }
+        }
+        // This shouldn't be possible.
+        // GenerateID() returns L"User.{}" unless it's invalid
+        return nullptr;

```

Comment:
This should be `{}`, not because it's more efficient (unlike Dustin's recent `return L""` change), but rather because C++23 removes the `nullptr` constructor for `std::string`.